sdk/messaging/azservicebus/ci.yml

```diff
@@ -0,0 +1,17 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
Please add the branch triggers like we did in https://github.com/Azure/azure-sdk-for-go/pull/15594 for the other pipelines.

Reply:
Done: https://github.com/Azure/azure-sdk-for-go/pull/15699/commits/89248b29071750ef84b135a518a85965640d2a2f